cmd/tikv-ctl/src/main.rs

```diff
@@ -2090,156 +1965,315 @@ fn main() {
         bottommost,
     } = cmd
     {
+        let pd = opt.pd.unwrap_or_else(|| {
+            clap::Error {
+                message: String::from("--pd is required for this command"),
+                kind: ErrorKind::MissingRequiredArgument,
+                info: None,
+            }
+            .exit();
+        });
```

Comment:
This seems to duplicate. Personally, I recommend extracting this out. For example:

```rust
if matches!(cmd, Cmd::CompactCluster { .. } | Cmd::SplitRegion { .. }) {
    // check pd
}
```

Reply:
It makes sense. I will fix it, thanks